### ISSUE TITLE
Unknown mod loader in manifest: 32.0.106

### ISSUE THREAD

**Issue by JakeMoe** [MultiMC/MultiMC5#3277]
System Information
-----------------------------
MultiMC version: 0.6.11-1344

Operating System: Debian 10 (Buster)

Summary of the issue or suggestion:
----------------------------------------------
When loading a custom modpack I'm creating (from zipfile), I get the error: `Unknown mod loader in manifest: 32.0.106`

After clicking OK, I'm able to Install Forge and select version 32.0.106, and the pack loads as expected.

What should happen:
------------------------------
Pack zipfile should load properly without error.

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. Create instance
2. Select zipfile (attached)

Suspected cause:
---------------------------
Unknown

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)

Additional Info:
---------------------------
[Primitus-0.1.zip](https://github.com/MultiMC/MultiMC5/files/5044595/Primitus-0.1.zip)

**Comment by peterix** [comment]
It says exactly what's wrong.

**Comment by peterix** [comment]
And no, this shouldn't work.

**Comment by JakeMoe** [comment]
Why shouldn't 32.0.106 work from manifest but work fine from a manual install?

**Comment by peterix** [comment]
Because it doesn't identify any known mod loader.

**Comment by JakeMoe** [comment]
Ok, reading between the lines, I think you're trying to tell me that Forge version != FML version?  Previously (1.15.2 and before), I'd just put the Forge version I wanted there and MutliMC used it fine.  So I went to CurseForge to find out how current packs are defining the FML version in their manifest (and the Forge version, if they're separate), and I don't think I like what I'm seeing.  None of them seem to have "Download" links for their client anymore, just "Install".  Servers have Download, but they don't typically run in a modded client (like MultiMC).  Have they well and truly screwed Linux users then?  Are they forcing the use of the Twitch client?

**Comment by peterix** [comment]
There is nothing between any lines.

`32.0.106` is a meaningless piece of text that doesn't identify anything.

**Comment by peterix** [comment]
Where did you get this pack?

**Comment by phit** [comment]
probably by hand

@JakeMoe  you are missing the `forge-` prefix look at some other new packs on curseforge and their manifest, e.g. https://i.imgur.com/K9G2xHL.png

**Comment by JakeMoe** [comment]
Oh my, thank you so much @phit, that was indeed it.  I even had a look at previous modpacks I've created, and that forge- prefix was there.  I've added the prefix and rebuild the pack, and it's loading fine in MultiMC now.

And yes, I was creating it by hand.  I don't have a choice; the Twitch app they want you to make packs with isn't available on Linux.  So I create them by hand, and I use MultiMC to play.
